var/spack/repos/builtin/packages/cpuinfo/package.py

```diff
@@ -10,11 +10,12 @@ class Cpuinfo(CMakePackage):
     """cpuinfo is a library to detect essential
     for performance optimization information about host CPU."""
 
-    homepage = "https://github.com/Maratyszcza/cpuinfo/"
-    git = "https://github.com/Maratyszcza/cpuinfo.git"
+    homepage = "https://github.com/pytorch/cpuinfo"
+    git = "https://github.com/pytorch/cpuinfo.git"
```

Comment:
https://github.com/Maratyszcza/cpuinfo is just a fork of https://github.com/pytorch/cpuinfo. PyTorch used to use the fork, and now uses upstream. Commits seems to exist in both repos.